src/dspbase.jl

```diff
@@ -146,7 +146,7 @@ function _conv(u, v, npad, nu, nv)
 end
 
 """
-    conv(u,v)
```

Comment:
In docstrings, four-space indents cause whatever follows the indent to be displayed as code. This indentation was intentional.